sirepo/feature_config.py

```diff
@@ -121,6 +121,9 @@ def b(msg, dev=False):
     i = _cfg.proprietary_sim_types.intersection(_cfg.default_proprietary_sim_types)
     assert not i, \
         f'{i}: cannot be in proprietary_sim_types and default_proprietary_sim_types'
+    _cfg.all_proprietary_sim_types = frozenset(
```

Comment:
`_cfg` should be restricted to pkconfig only, since the object should be treated as an abstraction. It should not be automatically assumed it is a PKDict (indeed it didn't used to be) and could just be augmented/edited to. If it is not something configurable, store elsewhere. It could be a function on feature_cfg.auth_controlled_sym_types.

Reply:
https://github.com/radiasoft/sirepo/pull/3573